src/main/java/com/google/sps/datastore/GetDatastore.java

```diff
@@ -0,0 +1,74 @@
+package com.google.sps.datastore;
+
+import java.util.ArrayList;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+
+//PCAP storage class
+public class GetDatastore {
```

Comment:
Usually a class name is a Noun. So something like PCAPDataStore? I am not sure what this class is trying to achieve so add some comments. If someone instantiates this they can get a list of PCAPdata objects per file or for all files?

Reply:
Make this an interface, something like `GenericPCAPDao` which has methods like getPCAPObjects(id).. you can then implement the interface with your datastore specific implementation.

Reply:
Changed class/method names